src/fragments/lib/storage/ios/download.mdx

```diff
@@ -4,6 +4,51 @@ There are three ways of getting data that was previously uploaded:
 
 You can download to in-memory buffer [Data](https://developer.apple.com/documentation/foundation/data) object with `Amplify.Storage.downloadData`:
 
+#### With StoragePath
```

Comment:
We should add a short blurb about how this is new in the latest version and we recommend customers to use this.

Reply:
I've added a callout for each of the new API.  I didn't specify a specific Amplify version, but we can make that update if you think it's necessary and when we have the exact version.